src/Components/UserProfile/UserProfile.js

```diff
@@ -3,44 +3,31 @@ import { useQuery } from "@apollo/client";
 import { GET_USER_BY_ID } from "../../queries";
 import "./UserProfile.css";
 import EventModal from '../EventModal/EventModal'
-import EventCard from "../EventCard/EventCard";
+import Events from "../Events/Events"
 
 // SETUP AS A FAMILY VIEW FROM THE EVENT DETAILS PAGE
 // Can use a query hook for data for DRY code or just pass as props
 
 const UserProfile = () => {
+  const { loading, error, data } = useQuery(GET_USER_BY_ID, {
+    variables: {"id": process.env.REACT_APP_USER_ID}
```

Comment:
I cant believe how simple this looks once the query is all set up this is awesome!